samples/boards/nrf/system_off/sample.yaml

```diff
@@ -22,9 +23,10 @@ tests:
     extra_configs:
       - CONFIG_APP_USE_NRF_RETENTION=y
   sample.boards.nrf.system_off.retained_mem:
-    extra_args: DTC_OVERLAY_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_retained_mem.overlay"
+    extra_args: DTC_OVERLAY_FILE="boards/nrf54l15dk_nrf54l15_cpuapp_retained_mem.overlay"
     build_only: true
     platform_allow:
+      - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
```

Comment:
Would't that cause using overlay for DK when building for PDK with twister? Maybe PDK should be removed here from `platform_allow` as in UARTE tests?

Reply:
In this case, I could resign from support PDK. 
@nordic-piks, do you agree with them?